ContentFiles/Forms/FormsUIContent/HomeView.xaml.cs.pp

```diff
@@ -0,0 +1,12 @@
+﻿using MvvmCross.Forms.Views;
+
+namespace $rootnamespace$.Views
+{
+    public partial class HomeView : MvxContentPage
```

Comment:
Can we use the strong typed version here? Something like:

`public partial class HomeView : MvxContentPage<HomeViewModel>`

Reply:
What's the advantage to doing this? Happy to change it if you think it's better to include it, but am not sure why, since it adds more things you have to add to the XAML file (trying where possible to have this be the briefest amount of changes for the user as possible)